k8s/dynatrace-gcp-function.yaml

```diff
@@ -95,6 +98,16 @@ spec:
             configMapKeyRef:
               name: dynatrace-gcp-function-config
               key: IMPORT_ALERTS
+        - name: HTTP_PROXY
+            valueFrom:
```

Comment:
There is incorrect indentation before 'valueFrom' - it should be the same as in the line above.